cmd/chore_copyright_update.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2022 OWASP Core Rule Set Project
+// SPDX-License-Identifier: Apache-2.0
+
+package cmd
+
+import (
+	"strconv"
+	"time"
+
+	"github.com/spf13/cobra"
+
+	"github.com/coreruleset/crs-toolchain/chore"
+	"github.com/coreruleset/crs-toolchain/context"
+)
+
+var choreCopyrightUpdateCmd = createChoreCopyrightUpdateCommand()
+var copyrightVariables struct {
+	Version string
+	Year    string
+}
+
+func init() {
+	buildChoreCopyrightUpdateCommand()
+}
+
+func createChoreCopyrightUpdateCommand() *cobra.Command {
+	return &cobra.Command{
+		Use:   "copyright-update",
+		Short: "Updates the copyright on every rule file",
+		Run: func(cmd *cobra.Command, args []string) {
+			rootContext := context.New(rootValues.workingDirectory.String(), rootValues.configurationFileName.String())
+			chore.CopyrightUpdate(rootContext, copyrightVariables.Version, copyrightVariables.Year)
+		},
+	}
+}
+
+func buildChoreCopyrightUpdateCommand() {
+	choreCmd.AddCommand(choreCopyrightUpdateCmd)
+	choreCopyrightUpdateCmd.Flags().StringVarP(&copyrightVariables.Year, "year", "y", strconv.Itoa(time.Now().Year()), "Year, example: 2023. Default: current year.")
+	choreCopyrightUpdateCmd.Flags().StringVarP(&copyrightVariables.Version, "version", "v", "CRS v10.0.1", "Add this text as the version to the file.")
```

Comment:
The version must be required, or you need to figure out the current version automatically.

Reply:
We can add both. Let me think about something.

Reply:
Right now I think the easy way is to require. We can add more things like reading the git repo in the future.